data/cwisep_j181006.00-101001.1.json

```
@@ -0,0 +1,77 @@
+{
+    "Sources": [
+        {
+            "source": "CWISEP J181006.00-101001.1",
+            "ra": 272.52574999999996,
+            "dec": -10.166805555555555,
+            "epoch": null,
+            "equinox": null,
+            "shortname": null,
+            "reference": "Schn20",
+            "other_references": null,
+            "comments": null
+        }
+    ], 
+    "ModeledParameters": [
+        {
+            "parameter": "log g",
+            "value": 5.0,
+            "value_error": 0.25,
+            "unit": "dex",
+            "comments": null,
+            "reference": "Lodi22"
+        },
+        {
+            "parameter": "mass",
+            "value": 17.0,
+            "value_error": 56.0,
+            "unit": "M_jup",
+            "comments": null,
```

Let's add the asymmetric uncertainty in the comment.  
```suggestion
            "comments":"12[+56, -12]"
```

---

should be 17.

---

Let's also make error null.